module2/index.js

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+import { v4 } from 'uuid';
+
+import { createUser, findUserById, getAutoSuggestUsers } from './users';
+import { validator } from './validation';
+
+const app = express();
+const router = express.Router();
+
+app.listen(3000, () => console.log('Server is running'));
+app.use(express.json());
+app.use('/', router);
+
+router.route('/users')

```

Comment:
Better to have this routers in a separate folder. One folder for one domain (like routers/users), one file for one route (/routers/users/getUser). Just because real world app logic on each route could be huge and it will be difficult to maintain your backed when all logic is in one file.

Reply:
Done